relay/README.md

```diff
@@ -150,12 +150,24 @@ message CircuitRelay {
     repeated bytes addrs = 2; // peer's known addresses
   }
 
-  optional Type type = 1;     // Type of the message
+  message Request {
+    optional Type type = 1;     // Type of the message
 
-  optional Peer srcPeer = 2;  // srcPeer and dstPeer are used when Type is HOP or STATUS
-  optional Peer dstPeer = 3;
-
-  optional Status code = 4;   // Status code, used when Type is STATUS
+    optional Peer srcPeer = 2;  // srcPeer and dstPeer are used when Type is HOP or STATUS
+    optional Peer dstPeer = 3;
+  }
+  
+  message Response {
+    optional Type type = 1;     // Type of the message
```

Comment:
We might don't need `Type` in `Response` messages anymore, but we **do** for `Request`.